### PR TITLE
Docs: true/false with boolean values in docsite/rst/os_guide

### LINKED ISSUE #78923: Docs:os_guide: replace yes/no with true/false for booleans
### Summary

Based on the [steering committee vote to use true/false for booleans,](https://github.com/ansible-community/community-topics/discussions/120) we are going through the guides to adapt to this change.

This issue requests these changes in the `docs/docsite/rst/os_guide/` files.

Changes are: change `yes` to `true`  and `no` to `false`

must be lowercase. Please open one PR to handle thse changes. It should impact 15 files. NOTE - ansibot does not like PRs over 50 files.

The following grep will help you find these occurrences from the `docs/docsite/rst/os_guide/` directory:
`grep -R '\: \(yes\|no\)$' --exclude-dir=locales`

### Issue Type

Documentation Report

### Component Name

docs/docsite/rst/os_guide.index.rst

### Ansible Version

```console
$ ansible --version
none
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
none
```

### OS / Environment

none

### Additional Information

none

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by sudiptog81**
##### SUMMARY
Changes yes/no in os_guide documentation to true/false based on the recommendations per the [steering committee vote to use true/false for booleans](https://github.com/ansible-community/community-topics/discussions/120).

Fixes #78923.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs/docsite/rst/os_guide/*.rst

### CODE REVIEW

File: docs/docsite/rst/os_guide/windows_dsc.rst [hunk 1]
```diff
@@ -205,10 +205,10 @@ For example, to define a ``[CimInstance]`` value in Ansible:
 
     # [CimInstance]AuthenticationInfo == MSFT_xWebAuthenticationInformation
     AuthenticationInfo:
-      Anonymous: no
-      Basic: yes
-      Digest: no
-      Windows: yes
+      Anonymous: false
```
Comment (Akasurde) [inline docs/docsite/rst/os_guide/windows_dsc.rst:208]: These values are for Windows-specific configuration and are not related to Ansible. Please check respective Windows documentation if these values accept True/False.

**Review by Akasurde — DISMISSED** [review]

File: docs/docsite/rst/os_guide/windows_dsc.rst (hunk 1 above)
Comment (sudiptog81) [reply docs/docsite/rst/os_guide/windows_dsc.rst:208]: Noted. As per some relevant code here, it seems Windows supports true/false, let me know what you think.

https://github.com/dsccommunity/WebAdministrationDsc/blob/e479b30fb9de6bd6778a98adfad20a3b3401210f/source/DSCResources/DSC_WebApplication/DSC_WebApplication.psm1#L590

Comment (Akasurde) [reply docs/docsite/rst/os_guide/windows_dsc.rst:208]: ok

**Review by Akasurde — COMMENTED** [review]
LGTM,  will wait for the docs team to update.

**Comment by oraNod** [comment]
@sudiptog81 Thanks for the contribution. I noticed in a duplicate PR that there is an additional boolean value to change here: https://github.com/ansible/ansible/pull/78991/files#diff-c0af7cca509fef4fb2354aee0706c4474e6f42be6796d8783e68c1b53e79bd34R132

Can you please update ``intro_bsd.rst`` as well?

**Comment by sudiptog81** [comment]
Sure, will do that real quick.

**Review by oraNod — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @sudiptog81 for the Ansible docs fix and your help with Hacktoberfest! And welcome to the Ansible project!

**Pull request merged**